static/app/components/organizations/pageFilters/pageFilterPinButton.tsx

```diff
@@ -6,19 +6,26 @@ import {IconLock} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import PageFiltersStore from 'sentry/stores/pageFiltersStore';
 import {useLegacyStore} from 'sentry/stores/useLegacyStore';
-import {PinnedPageFilter} from 'sentry/types';
+import {Organization, PinnedPageFilter} from 'sentry/types';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
 
 type Props = {
   filter: PinnedPageFilter;
+  organization: Organization;
```

Comment:
Should we just `useOrganization` instead of prop drilling?

Reply:
good call, will make that change